### PR TITLE
heroic-games-launcher: Add version 2.4.3

### LINKED ISSUE #501: (Request) add heroic-game-launcher
PackageIdentifier: Heroic
PackageVersion: 1.8.2
PackageLocale: en-US
Publisher: flavioislima
PackageName: Heroic-game-launcher
License: open source
ShortDescription: open source epic game client
Installers:

Architecture: x64
InstallerType: exe
InstallerUrl: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/v1.8.2/Heroic.Setup.1.8.2.exe
ManifestType: singleton
ManifestVersion: 1.0.0

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #501
Relates to #504

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/heroic-games-launcher.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "2.4.3",
+    "description": "A native GOG and Epic Games Launcher GUI for Linux, Windows and Mac.",
```
Comment (Calinou) [inline bucket/heroic-games-launcher.json:3]:
```suggestion
    "description": "A native GOG and Epic Games Launcher GUI",

```

File: bucket/heroic-games-launcher.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "2.4.3",
+    "description": "A native GOG and Epic Games Launcher GUI",
```
Comment (Calinou) [inline bucket/heroic-games-launcher.json:3]:
```suggestion
    "description": "Native GUI reimplementation for GOG and Epic Games Launcher",

```

File: bucket/heroic-games-launcher.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.4.3",
+    "description": "Native GUI reimplementation for GOG and Epic Games Launcher",
```
Comment (Calinou) [inline bucket/heroic-games-launcher.json:3]:
```suggestion
    "description": "Unofficial native GUI reimplementation for GOG and Epic Games Launcher",

```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by Zliced13** [comment]
Your welcome, glad I could help.

**Pull request merged**